docs/host_install.md

```diff
@@ -176,6 +176,14 @@ running Private Data Objects. It is recommended that Sawtooth be run on Ubuntu
 16.04 as it is the only operating system version on which Sawtooth is actively
 supported.
 
+There is an ongoing effort to support Microsoft CCF based ledger
+in addition to Hyperledger Sawtooth. Currently PDO supports CCF ledger under
+the SIMULATE mode for PDO enclaves, and virtual mode for CCF enclaves.
```

Comment:
By "SIMULATE" mode, you mean SGX simulation mode?

Reply:
yes, thank you!